### PR TITLE
Robch/2404 apr29 expect not expect regex rename

### PR DESCRIPTION
**Pull request by robch**

### CODE REVIEW

File: src/ai/.x/templates/helper_functions/UserNameHelperFunctionClass.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 using Azure.AI.Details.Common.CLI.Extensions.HelperFunctions;
 
-public static class {ClassName}
+public static class UserNameHelperFunctionClass
```
Comment (chschrae) [inline src/ai/.x/templates/helper_functions/UserNameHelperFunctionClass.cs:8]: I'm not clear on why this change is made. Don't all of the templates need to be updated if you take ClassName out like this?

**Review by chschrae — APPROVED** [review]

File: src/ai/.x/templates/helper_functions/UserNameHelperFunctionClass.cs (hunk 1 above)
Comment (robch) [reply src/ai/.x/templates/helper_functions/UserNameHelperFunctionClass.cs:8]: I'm still working out how the class name even works in the other ones for people to "redefine"... It might go away. This one here was problematic in that, the "AI", when making a new helper function class, often chose the same simple name as this one (as configured by default). Thus, this one was changed to have a unique name that reflects the single helper function that it has (getting a user name).
